staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -185,6 +185,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	RemoveSelfLink:           {Default: true, PreRelease: featuregate.Beta},
 	SelectorIndex:            {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
 	WarningHeaders:           {Default: true, PreRelease: featuregate.Beta},
-	EfficientWatchResumption: {Default: false, PreRelease: featuregate.Alpha},
+	EfficientWatchResumption: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
update the comment for the `EfficientWatchResumption` featuregate around like 153

Reply:
done